packages/components/pie-webc/src/componentService.js

```diff
@@ -0,0 +1,132 @@
+import chalk from 'chalk';
+
+export class ComponentService {
```

Comment:
More of a general question, how do we feel about using (or not) TypeScript for utility/script code, like this class?

Reply:
I'm not agains the idea in principle. I think typings for the node `path` and `fs` modules could be tricky (or very easy) to set up. Having typescript would also mean we have to run the typescript compiler before `npx` can run, I believe.